config/default.yaml

```diff
@@ -2,18 +2,19 @@ proxy:
   aggregated: http://localhost:4452
   data: http://localhost:4452
   definition_import: http://localhost:4451
-  document_management: http://localhost:4603
+  document_management: http://localhost:4506
```

Comment:
for my knowledge, how come these changes in port the numbers in this file?

Reply:
Well, they should have been fixed before, to run the gateway outside a docker container. We had to fix these to make it work.